tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1594,6 +1594,7 @@ public int compare(Map<String, Object> o1, Map<String, Object> o2) {
         }
         XMLSource diskData = sm.getDiskFactory().makeSource(locale.getBaseName()).freeze(); // trunk
         CLDRFile cldrFile = fac.make(loc, true, true);
+        CLDRFile cldrUnresolved = cldrFile.getUnresolved();
```

Comment:
i know this is still a draft pr -- do the cldrUnresolved and conn.commit() parts belong in different tickets?

Reply:
The unresolved part is already in a separate commit… This has become a grab bag of improvements, I intend to split it out to separate PRS

Reply:
conn.commit()  is a separate issue.